src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -66,11 +67,18 @@ public static IEnumerable<object[]> SslStream_StreamToStream_Authentication_Succ
             }
         }
 
-        [Theory]
+        [ConditionalTheory]
```

Comment:
Why `ConditionalTheory` if there are no actual conditions passed to it?

Reply:
That's necessary for `SkipTestException` to work. The condition isn't statically defined but dynamically via throwing an Exception.

Reply:
BTW this is our own extension and that is also why Microsoft.DotNet.XUnitExtensions is needed @antonfirsov